addons/default/visiosoft/advs-module/resources/views/list/partials/ads.twig

```diff
@@ -5,16 +5,13 @@
     <table class="w-100 text-center">
         <thead>
         <tr class="text-white">
-            <th class="text-nowrap" colspan="2">{{ trans('visiosoft.module.advs::field.ad_title') }}</th>
+            <th class="text-nowrap" colspan="3">{{ trans('visiosoft.module.advs::field.ad_title') }}</th>
```

Comment:
why make it 3? it should extend only 2 columns, the photo and the title columns

Reply:
@diashalabi #3835 -12 check Gitlab issue

Reply:
if one customer wants to remove it, that doesn't mean all customers want to remove it